app/controllers/webhooks/youtube_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Webhooks
+  class YoutubeController < ApplicationController
+    include CustomControllerHelpers
+
+    before_action :check_linked_account_param
+
+    def verify
+      mode = params['hub.mode']
+      topic = params['hub.topic']
+      challenge = params['hub.challenge']
+
+      if !linked_account && mode == 'unsubscribe' # deleted linked-account
```

Comment:
This whole if statement is kinda messy. Not sure if there is a way to make it nicer though.

Reply:
Yeah pubsubhubbub doesn't lend itself to clean code tbh